server/container/cli/src/test/java/org/apache/james/cli/type/CmdTypeTest.java

```diff
@@ -139,6 +139,12 @@ public void lookupDeleteMailboxShouldReturnEnumValue() {
     }
 
     @Test
+    public void lookupImportEmlFileToMailboxShouldReturnEnumValue() {
+        assertThat(CmdType.lookup("ImportEmlFileToMailbox"))
```

Comment:
hmm, what is your last decision for that :) should i change the name "importEmlFileToMailbox" as importEml for all occurences